LongReadRNABenchmark/bambu_docker/Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get -qq update && apt-get -qq -y install \
```

Comment:
are all of these packages really needed in each of these dockers?  I suspect texlive for example might not be necessary...

Reply:
Significantly cleaned up the dockerfiles. Most, if not all, unnecessary packages should be removed now. Resolved.